modules/server/src/utils/getAllData.js

```diff
@@ -6,7 +6,7 @@ function getAllData({
   projectId,
   index,
   variables = null,
-  size = 10000,
+  chunkSize = 1000,
```

Comment:
env var?

Reply:
or possibly even something configurable at the `arranger-projects` level? would be a lot easier to test chunk sizes that way

Reply:
I was going to make it a param, so if you had a particular large one that was timing out you could set it from the point of use. but then I didn't. 
what do you think of that? otherwise I will just make an env for now :D

Reply:
param is a great idea

Reply:
made it a param 👍